projects/components/src/modal/modal-container.component.scss

```diff
@@ -17,8 +17,8 @@
   }
 
   &.modal-size-medium {
-    height: 456px;
-    width: 530px;
+    height: 530px;
```

Comment:
@jake-bassett you added this size for your use case right? did you need it in the other orientation?

Reply:
This size works for what I'm doing too.